src/promptflow/promptflow/executor/_prompty_executor.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from typing import Optional
+
+from promptflow.contracts.tool import InputDefinition
+from promptflow.core._flow import Prompty
+from promptflow.storage import AbstractRunStorage
+from promptflow.tracing._trace import _traced
+
+from ._script_executor import ScriptExecutor
+
+
+class PromptyExecutor(ScriptExecutor):
```

Comment:
add some docstring describe the dfiferent with PromptyExecutor & ScriptExecutor

Reply:
add docstring